src/app/auth/auth.service.ts

```diff
@@ -179,4 +200,45 @@ var ProductName = sessionStorage.getItem('ProductName');
             return fetchresult;
         }));
     }
+
+    captchaVerify(token: string){
+
+
+ 
+
+       let httpOptions1 = {
+        headers: new HttpHeaders({
+            'Access-Control-Allow-Methods':'DELETE, POST, GET, OPTIONS',
+            'Access-Control-Allow-Headers':'Content-Type, Access-Control-Allow-Headers, Authorization, X-Requested-With',
+            'Content-Type':  'application/x-www-form-urlencoded',
+            'Access-Control-Allow-Origin':'http://localhost:4200'
+        })
+      };
+
+      this.headers1 = new HttpHeaders(
+        {
+            'Content-Type':  'application/x-www-form-urlencoded',
+            Accept: '*/*',
+
+            
+        }
+    );
+
+
+
+
+let data={
+//6LeJqqsZAAAAADgHTJGWg4YMI028ffU6c--Av2YR(client)
+//6LdoiLwZAAAAADmQpBrZnki6eFbWJS-WaD1r1luU---owner
+//secret:"6LdoiLwZAAAAADmQpBrZnki6eFbWJS-WaD1r1luU",
+
+   secret:"6LdoiLwZAAAAADmQpBrZnki6eFbWJS-WaD1r1luU",
+   response:token
+}
+
+      const body = new HttpParams({fromObject: data});
+    const options = { headers: this.headers1};
+    return this.http.post(`https://www.google.com/recaptcha/api/siteverify`, body.toString(), options);
```

Comment:
@Bithika-Redapple Do not hardcode this url value. Retrieve the url from a Constants